### ISSUE TITLE
Needs to be patched for kde 6.1

### ISSUE THREAD

**Issue by EliteAMDGamer** [taj-ny/kwin-effects-forceblur#83]
No longer showing up in window management with aur reinstall and manual build fails.

```
[  4%] Automatic MOC for target forceblur
AutoMoc warning
---------------
"SRC:/src/main.cpp"
includes the moc file "main.moc", but does not contain a Q_OBJECT, Q_GADGET, Q_NAMESPACE, Q_NAMESPACE_EXPORT, Q_GADGET_EXPORT, Q_ENUM_NS, K_PLUGIN_FACTORY, K_PLUGIN_CLASS, K_PLUGIN_FACTORY_WITH_JSON or K_PLUGIN_CLASS_WITH_JSON macro.

[  4%] Built target forceblur_autogen
[  9%] Building CXX object src/CMakeFiles/forceblur.dir/blur.cpp.o
/home/elite/kwin-effects-forceblur/src/blur.cpp: In member function ‘virtual void KWin::BlurEffect::reconfigure(KWin::Effect::ReconfigureFlags)’:
/home/elite/kwin-effects-forceblur/src/blur.cpp:199:47: warning: unused parameter ‘flags’ [-Wunused-parameter]
  199 | void BlurEffect::reconfigure(ReconfigureFlags flags)
      |                              ~~~~~~~~~~~~~~~~~^~~~~
/home/elite/kwin-effects-forceblur/src/blur.cpp: In member function ‘void KWin::BlurEffect::updateBlurRegion(KWin::EffectWindow*)’:
/home/elite/kwin-effects-forceblur/src/blur.cpp:271:28: warning: variable ‘borders’ set but not used [-Wunused-but-set-variable]
  271 |             const QMargins borders = w->decoration()->borders();
      |                            ^~~~~~~
/home/elite/kwin-effects-forceblur/src/blur.cpp: In static member function ‘static bool KWin::BlurEffect::supported()’:
/home/elite/kwin-effects-forceblur/src/blur.cpp:411:61: error: ‘supported’ is not a member of ‘KWin::GLFramebuffer’
  411 |     return effects->isOpenGLCompositing() && GLFramebuffer::supported() && GLFramebuffer::blitSupported();
      |                                                             ^~~~~~~~~
/home/elite/kwin-effects-forceblur/src/blur.cpp:411:91: error: ‘blitSupported’ is not a member of ‘KWin::GLFramebuffer’
  411 |     return effects->isOpenGLCompositing() && GLFramebuffer::supported() && GLFramebuffer::blitSupported();
      |                                                                                           ^~~~~~~~~~~~~
/home/elite/kwin-effects-forceblur/src/blur.cpp: In member function ‘void KWin::BlurEffect::blur(const KWin::RenderTarget&, const KWin::RenderViewport&, KWin::EffectWindow*, int, const QRegion&, KWin::WindowPaintData&)’:
/home/elite/kwin-effects-forceblur/src/blur.cpp:853:33: error: ‘class KWin::WindowPaintData’ has no member named ‘projectionMatrix’
  853 |         projectionMatrix = data.projectionMatrix();
      |                                 ^~~~~~~~~~~~~~~~
/home/elite/kwin-effects-forceblur/src/blur.cpp:895:48: error: ‘class KWin::WindowPaintData’ has no member named ‘projectionMatrix’
  895 |             QMatrix4x4 projectionMatrix = data.projectionMatrix();
      |                                                ^~~~~~~~~~~~~~~~
make[2]: *** [src/CMakeFiles/forceblur.dir/build.make:120: src/CMakeFiles/forceblur.dir/blur.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:714: src/CMakeFiles/forceblur.dir/all] Error 2
make: *** [Makefile:146: all] Error 2 
```

```
Operating System: Garuda Linux 
KDE Plasma Version: 6.1.0
KDE Frameworks Version: 6.3.0
Qt Version: 6.7.1
Kernel Version: 6.9.5-1-cachyos (64-bit)
Graphics Platform: Wayland
```

**Comment by kalwabed** [comment]
Hi, I wanted to tell you that I also faced this issue before. But, when I upgraded to version `v1.3.0` the _force-blur_ was normal again.
Btw, I'm using CachyOS with Plasma 6.1.

**Comment by EliteAMDGamer** [comment]
After changing the location for the build folder got it installed an working just fine

**Comment by D3SOX** [comment]
As this is a compiled C++ effect it's normal for it to be missing when you upgrade Plasma. It needs to be compiled against your Plasma version. Now that Arch released 6.1 to stable I've bumped the AUR package release so it should automatically be picked up as update and rebuilt by your AUR helper
